evil-collection-debbugs.el

```diff
@@ -49,7 +49,7 @@
     ;; filter
     (kbd "s") 'debbugs-gnu-narrow-to-status
```

Comment:
We can leave out the `kbd` here! :)

Reply:
I think it was there before, but I can remove it for sure.

Reply:
Yes, that was my mistake :p